rubix-bookkeeper/src/test/java/com/qubole/rubix/bookkeeper/TestBookKeeper.java

```diff
@@ -410,4 +410,11 @@ public void verifyCacheHitAndMissMetricsAreReported() throws TException
     assertEquals(metrics.getGauges().get(BookKeeperMetrics.CacheMetric.CACHE_HIT_RATE_GAUGE.getMetricName()).getValue(), 0.5);
     assertEquals(metrics.getGauges().get(BookKeeperMetrics.CacheMetric.CACHE_MISS_RATE_GAUGE.getMetricName()).getValue(), 0.5);
   }
+
```

Comment:
We don't need this test. This can be tested the test cases we are writing in rubix-test module

Reply:
Yeah. Removing this. 
Added this to test the behavior of codecov report